microhapulator/Snakefile

```diff
@@ -38,11 +44,17 @@ rule report:
         expand("analysis/{sample}/callplots/.done", sample=config["samples"]),
         expand("analysis/{sample}/{sample}-off-target-reads.csv", sample=config["samples"]),
         resource_filename("microhapulator", "data/template.html"),
+        resource_filename("microhapulator", "data/marker_details_template.html"),
```

Comment:
Added the new template file as a prerequisite for this rule. Helped simplify debugging: Snakemake will regenerate the report if the file has been updated since the report was last generated.